worlds/alttp/__init__.py

```diff
@@ -396,7 +396,16 @@ def stage_fill_hook(cls, world, progitempool, nonexcludeditempool, localrestitem
                     trash_count -= 1
 
     def get_filler_item_name(self) -> str:
-        return "Rupees (5)"  # temporary
+        if self.world.goal[self.player] == "icerodhunt":
+            item = "Nothing"
+        else:
+            extras_list = difficulties[self.world.difficulty[self.player]].extras[0] \
+            + difficulties[self.world.difficulty[self.player]].extras[1] \
+            + difficulties[self.world.difficulty[self.player]].extras[2] \
+            + difficulties[self.world.difficulty[self.player]].extras[3] \
+            + difficulties[self.world.difficulty[self.player]].extras[4]
+            item = self.world.random.choice(extras_list)
```

Comment:
Could perhaps be shortened to;

```suggestion
            extras_list = difficulties[self.world.difficulty[self.player]].extras[0:4]
            item = self.world.random.choice(extras_list)
```

Reply:
this does not seem like it should produce the same result:
1. the `+` in the original code merges the lists
2. `[0:4]` does not include `[4]`

Reply:
Good catch, my off the cuff python is rusty. 

Updated suggestion:

from itertools import chain
---
extras_list = chain(difficulties[self.world.difficulty[self.player]].extras[0:5]

Either way, I believe that snippet can be improved and I think we might as well if we can.

Reply:
I don't understand this chain thing and what you wrote alone did not work